MQTT433gateway.ino

```diff
@@ -59,7 +61,13 @@ boolean rawMode = false;
 String otaURL = "";
 
 void setup() {
-  Serial.begin(115200);
+  if (RECEIVER_PIN==3) {
+    Serial.begin(115200,SERIAL_8N1,SERIAL_TX_ONLY); 
```

Comment:
Only this line is enough, to add ESP-01 support. A if condition is not necessary, because MQTT433gateway did not read from serial.